plugins/reports/frontend/public/javascripts/countly.views.js

```diff
@@ -33,6 +32,9 @@
             };
         },
         methods: {
+            createReport: function() {
+                $("#create-report-button").click();
```

Comment:
why are we using still jquery in reports? 😄

Reply:
fixed